Makefile

```diff
@@ -294,7 +294,7 @@ gfortran:
 	"CC_SERIAL = gcc" \
 	"CXX_SERIAL = g++" \
 	"FFLAGS_PROMOTION = -fdefault-real-8 -fdefault-double-8" \
-	"FFLAGS_OPT = -O3 -m64 -ffree-line-length-none -fconvert=big-endian -ffree-form" \
+	"FFLAGS_OPT = -O3 -m64 -ffree-line-length-none -fconvert=big-endian -ffree-form -fallow-argument-mismatch -fallow-invalid-boz" \
```

Comment:
It might be good to ask @mgduda what the implications of are for adding these two flags.  @metdyn, do we already use these flags in the cmake build?

Reply:
For what it's worth, the current MPAS v7.3 release doesn't require the `-fallow-argument-mismatch -fallow-invalid-boz` flags when building with the GNU 9, 10, and 11 compilers. See, e.g., PR MPAS-Dev/MPAS-Model#736 .

Reply:
Thank you Michael!  @metdyn, could you apply that same change in this branch, then remove the change to Makefile?  Eventually we need to merge MPASv7.3 into our fork(s).

Reply:
@mgduda:  that is a good fix:
      integer, parameter:: bigint = int(O'17777777777')   
I will make corresponding changes to this set of code deposit.  Thanks for letting us know!